exercises/practice/killer-sudoku-helper/src/test/java/KillerSudokuHelperTest.java

```diff
@@ -0,0 +1,57 @@
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class KillerSudokuHelperTest {
+
+    private KillerSudokuHelper helper = new KillerSudokuHelper();
+
+    @Test
+    public void testTrivialOneDigitCages() {
+        for (int n = 1; n <= 9; n++) {
+            List<List<Integer>> expected = List.of(List.of(n));
+            assertEquals(expected, helper.combinationsInCage(n, 1));
```

Comment:
[We prefer using AssertJ assertions over JUnit assertions](https://github.com/exercism/java/blob/main/POLICIES.md#prefer-assertj-assertions). Could you please update the tests to use AssertJ instead?